src/posts/wearethecity-shortlist-2016-entrepreneur-category.md

```diff
@@ -16,7 +16,9 @@ It’s been another exciting start to the week. I’ve been shortlisted as co-fo
 
 The WATC Rising Star Awards are the first to focus on the UK’s female talent pipeline below management level and celebrates 100 individual female contributors representing the leaders and role models of tomorrow.
 
+<p class="pullquote">
```

Comment:
need to differentiate between pullquotes that repeat text or pull out text chunks to highlight as an aside and therefore need to stay in the content on mobile.